pyCGM_Single/pycgmStatic.py

```diff
@@ -529,49 +814,54 @@ def hipJointCenter(frame,pel_origin,pel_x,pel_y,pel_z,vsk=None):
     return hip_JC
     
 def hipAxisCenter(l_hip_jc,r_hip_jc,pelvis_axis):
-    
-    """
-
-    Calculate the hip joint axis function.
+    """Calculate the hip joint axis function.
 
     Takes in a hip joint center of x,y,z positions as well as an index.
     and takes the hip joint center and pelvis origin/axis from previous functions.
     Calculates the hip axis and returns hip joint origin and axis.
-    -------------------------------------------------------------------------
 
-    INPUT:  Array of R_hip_jc, L_hip_jc, pelvis_axis each x,y,z position.
-            and pelvis_axis is array of pelvis origin and axis. the axis also composed of 3 arrays 
-            each things are x axis, y axis, z axis.
-            
-    OUTPUT: Returns the hip Axis Center and hip Axis.
-             return = [[hipaxis_center x,y,z position],
-                       [array([[hipaxis_center x,y,z position],
-                               [hip x_axis x,y,z position]]),
+    Hip center axis: Computed by taking the mean at each x,y,z axis of the left and right hip joint center.
+    Hip axis: Computed by getting the summation of the pelvis and hip center axes.
+
+    Parameters
+    ----------
+    l_hip_jc, r_hip_jc: array
+        Array of R_hip_jc and L_hip_jc each x,y,z position.
+    pelvis_axis : array
+        An array of pelvis origin and axis. The axis is also composed of 3 arrays,
+        each things are x axis, y axis, z axis.
+       
+    Returns
+    -------
+    array
+        Returns the hip Axis Center and hip Axis.
+            return = [[hipaxis_center x,y,z position],
```

Comment:
I think the formatting here could use some work.

Reply:
Fixed the formatting to be more clear